src/TemplateProvider.php

```diff
@@ -124,13 +138,13 @@ protected static function curl_call($url, $connectTimeoutMs = 100, $timeoutMs =
 
 		if($response === false || curl_errno($ch)) {
 			$message = 'Could not fetch toolbar from '.$url.': ' . curl_error($ch);
-			SS_Log::log(new Exception($message, 500), SS_Log::ERR);
+			Log::log(new Exception($message, 500), Log::ERR);
```

Comment:
These calls should use `Injector::inst()->get('Logger')` then te appropriate log level since we have monolog now